pkg/config/config_template.yaml

```diff
@@ -1,858 +1,1197 @@
+
+## @param process_config - custom object - optional
+## Enter specific configurations for your Process data collection.
+## Uncomment this parameter and the one below to enable them.
+## See https://docs.datadoghq.com/graphing/infrastructure/process/
 #
+# process_config:
+
+  ## @param enabled - string - optional - default: "false"
+  ##  A string indicating the enabled state of the Process Agent:
+  ##    * "false"    : The Agent collects only containers information.
+  ##    * "true"     : The Agent collects containers and processes information.
+  ##    * "disabled" : The Agent process collection is disabled.
+  #
+  # enabled: "true"
+
+  ## @param expvar_port - string - optional - default: 6062
+  ## Port for the debug endpoints for the process-agent.
+  #
+  # expvar_port: 6062
+
+  ## @param log_file - string - optional
+  ## The full path to the file where process-agent logs are written.
+  #
+  # log_file: <PROCESS_LOG_FILE_PATH>
+
+  ## @param intervals - custom object - optional - default: 10s for normal checks and 2s for others.
+  ## The interval, in seconds, at which the Agent runs each check. If you want consistent
+  ## behavior between real-time, set the `container_realtime` and `process_realtime` intervals to 10.
+  #
+  # intervals:
+  #   container: 10
+  #   container_realtime: 2
+  #   process: 10
+  #   process_realtime: 2
+
+  ## @param blacklist_patterns - list of strings - optional
+  ## A list of regex patterns that exclude processes if matched.
+  #
+  # blacklist_patterns:
+  #   - <REGEX>
+
+  ## @param queue_size - integer - optional - default: XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+  ## How many check results to buffer in memory when POST fails.
+  #
+  # queue_size: XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+  ## @param max_proc_fds - integer - optional - default: XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+  ## The maximum number of file descriptors to open when collecting net connections.
+  ## Only change if you are running out of file descriptors from the Agent.
+  #
+  # max_proc_fds: XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+  ## @param max_per_message - integer - optional - default: XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+  ## The maximum number of processes or containers per message.
+  #
+  # max_per_message: XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+  ## @param dd_agent_bin - string - optional - default: XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+  ## Overrides the path to the Agent bin used for getting the hostname.
+  #
+  # dd_agent_bin: XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+  ## @param dd_agent_env - string - optional - default: XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+  ## Overrides of the environment we pass to fetch the hostname.
+  #
+  # dd_agent_env: XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+  ## @param scrub_args - boolean - optional - default: true
+  ## Hide sensitive data on the Live Processes page.
+  #
+  # scrub_args: true
+
+  ## @param custom_sensitive_words - list of strings - optional
+  ## Define your own list of sensitive data to be merged with the default one.
```

Comment:
Let's add a link to: https://docs.datadoghq.com/graphing/infrastructure/process/?tab=linuxwindows#process-arguments-scrubbing here as well